<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAudio</class>
 <widget class="QWidget" name="FormAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	color: rgb(255, 255, 255);
	border-width:3px;
	border-style:solid;
	border-color: rgb(9,183,198);
}
QPushButton{
	color: rgb(255, 255, 255);
	border:none;
	background-color: rgb(95, 95, 95);
}
QPushButton::focus{
	color:rgb(60, 60, 60);
	border:none;
	background-color: rgb(208, 230, 255);
}
QPushButton::disabled{
	color:rgb(0, 0, 0);
	border:none;
	background-color: rgb(226, 226, 226);
}
</string>
  </property>
  <widget class="QLabel" name="labelBG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/BK.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>470</width>
     <height>262</height>
    </rect>
   </property>
   <property name="title">
    <string>MP3</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>20</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>110</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>110</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>199</y>
      <width>110</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
